foxy/distribution.yaml

```diff
@@ -1600,6 +1600,21 @@ repositories:
       url: https://github.com/ros2/libyaml_vendor.git
       version: foxy
     status: maintained
+  map_transformer:
+    doc:
+      type: git
+      url: https://github.com/osrf/map_transformer.git
+      version: 1.0.0
```

Comment:
As I understand, Git tags aren't valid for for the version in `source` and `doc` entries. Prefer to point to branch where the release was made from (or where docs are living). I think should fix CI.

Reply:
Thanks for the tip. I've changed it to use the branch name.